humboldt-state-university-environmental-resources-engineering.csl

```diff
@@ -0,0 +1,181 @@
+<?xml version="1.0" encoding="utf-8"?>
+<style xmlns="http://purl.org/net/xbiblio/csl" class="in-text" version="1.0" demote-non-dropping-particle="sort-only" default-locale="en-US">
+  <info>
+    <title>Humboldt State University-Environmental Resources Engineering</title>
+    <title-short>hsu</title-short>
+    <id>http://www.zotero.org/styles/Humboldt State Univeristy-Environmental Resources Engineering</id>
```

Comment:
This needs to be `http://www.zotero.org/styles/humboldt-state-university-environmental-resources-engineering`